src/librustc_trans/back/write.rs

```diff
@@ -1012,6 +1012,9 @@ unsafe fn configure_llvm(sess: &Session) {
         if sess.time_llvm_passes() { add("-time-passes"); }
         if sess.print_llvm_passes() { add("-debug-pass=Structure"); }
 
+        // FIXME #21627 disable faulty FastISel on AArch64 (even for -O0)
```

Comment:
I think this number should refer to an open bug (i.e. you can preemptively open one "reenable FastISel on AArch64" before this lands) so that it gets picked up in future triage/general bug handling.